src/utils/parse_utils.ts

```diff
@@ -41,85 +44,82 @@ export const parseUtils = {
+            // schema.
+            if (request.takerAddress === undefined) {
+                throw new ValidationError([
+                    {
+                        field: 'takerAddress',
+                        code: ValidationErrorCodes.RequiredField,
+                        reason: ValidationErrorReasons.TakerAddressInvalid,
+                    },
+                ]);
+            }
 
-                // If the user is requesting a firm quote, we want to make sure that `intentOnFilling` is set to "true".
-                if (endpoint === 'quote' && request.intentOnFilling !== 'true') {
-                    throw new ValidationError([
-                        {
-                            field: 'intentOnFilling',
-                            code: ValidationErrorCodes.RequiredField,
-                            reason: ValidationErrorReasons.RequiresIntentOnFilling,
-                        },
-                    ]);
-                }
+            // We enforce a valid API key - we don't want to fail silently.
+            if (request.apiKey === undefined) {
+                throw new ValidationError([
+                    {
+                        field: '0x-api-key',
+                        code: ValidationErrorCodes.RequiredField,
+                        reason: ValidationErrorReasons.InvalidApiKey,
+                    },
+                ]);
+            }
+            if (!validApiKeys.includes(request.apiKey)) {
+                throw new ValidationError([
+                    {
+                        field: '0x-api-key',
+                        code: ValidationErrorCodes.FieldInvalid,
+                        reason: ValidationErrorReasons.InvalidApiKey,
+                    },
+                ]);
+            }
 
-                return {
-                    nativeExclusivelyRFQT: true,
-                    excludedSources: Object.values(ERC20BridgeSource).filter(t => t !== ERC20BridgeSource.Native),
-                };
-            } else {
+            // If the user is requesting a firm quote, we want to make sure that `intentOnFilling` is set to "true".
+            if (endpoint === 'quote' && request.intentOnFilling !== 'true') {
                 throw new ValidationError([
                     {
-                        field: 'includedSources',
-                        code: ValidationErrorCodes.IncorrectFormat,
-                        reason: ValidationErrorReasons.ArgumentNotYetSupported,
+                        field: 'intentOnFilling',
+                        code: ValidationErrorCodes.RequiredField,
+                        reason: ValidationErrorReasons.RequiresIntentOnFilling,
                     },
                 ]);
             }
+
+            return {
+                excludedSources: [],
+                includedSources: parseUtils.parseStringArrForERC20BridgeSources(['0x', ...includedIds]),
```

Comment:
`RFQT` is not part of ERC20Source, will it get removed when `parseStringArrForERC20BridgeSources` is called?

Reply:
Yes.